spec/features/admin_views_councillor_contribution_spec.rb

```diff
@@ -51,10 +51,25 @@
 
     click_link "View"
 
-    expect(page).to have_content "Contributor Felix Chaung"
-    expect(page).to have_content "Authority Casey City Council"
+    expect(page).to have_content "Felix Chaung"
+    expect(page).to have_content "example@gmail"
```

Comment:
Probably best to use the whole email address so it's clear what we're actually looking for, i.e. not just part of the email address but the whole thing.